app/Console/Commands/MercatorModulesList.php

```
@@ -0,0 +1,64 @@
+namespace App\Console\Commands;
+
+use App\Modules\ModuleDiscovery;
+use App\Modules\ModuleRegistry;
+use Illuminate\Console\Command;
+
+class MercatorModulesList extends Command
+{
+    protected $signature = 'mercator:modules:list';
+    protected $description = 'List of available Mercator modules and their status';
+
+    public function handle(ModuleDiscovery $discovery, ModuleRegistry $registry): int
```

PHPStan (2.1.32)

Parameter $discovery of method App\Console\Commands\MercatorModulesList::handle() has invalid type App\Modules\ModuleDiscovery.


(class.notFound)